MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -33,6 +35,14 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 		}
 	}
     
+    func display(_ viewModel: FeedErrorViewModel) {
+        if viewModel.hasMessage {
+            errorView?.show(message: viewModel.message!)
```

Comment:
You can avoid the force-unwrap with an `if let`:

```suggestion
        if let message = viewModel.message {
            errorView?.show(message: message)
```